### ISSUE TITLE
Email addresses with modifier break script

### ISSUE THREAD

**Issue by MDKAOD** [FailSpy/humble-steam-key-redeemer#39]
I use a humble account with a '+' modifier, this breaks the script with the following error:

```
Traceback (most recent call last):
  File "E:\humble-steam-key-redeemer-main\humblesteamkeysredeemer.py", line 907, in <module>
    humble_login(driver)
  File "E:\humble-steam-key-redeemer-main\humblesteamkeysredeemer.py", line 263, in humble_login
    username = input("Humble Email: ")
               ^^^^^^^^^^^^^^^^^^^^^^^
EOFError: EOF when reading a line
```

Probably should sanitize the string

Additional information:

```
Traceback (most recent call last):
  File "E:\humble-steam-key-redeemer-main\humblesteamkeysredeemer.py", line 916, in <module>
    export_mode(driver,order_details)
  File "E:\humble-steam-key-redeemer-main\humblesteamkeysredeemer.py", line 752, in export_mode
    best_match = match_ownership(owned_app_details,tpk)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: match_ownership() missing 1 required positional argument: 'filter_live'
```

**Comment by FailSpy** [comment]
Dang, never thought to test that. Sorry you hit up against this, will look into a fix.

**Comment by MDKAOD** [comment]
I added another comment above. I'm sure sanitizing the input is all that's needed. I'm pretty inexperienced, but likely just need a statement that checks for + modifiers (google uses them to filter email see [here](https://www.androidauthority.com/gmail-tips-and-tricks-3319527/).

Check Stack Overflow for a hint: https://stackoverflow.com/questions/32054891/python-input-sanitization

**Comment by MDKAOD** [comment]
Batch script through Windows shell (CMD)

**Comment by MDKAOD** [comment]
Appears to be working.. I guess the second error corrected whatever was wrong with error #1
